test/types/bytes/serialization.chpl

```diff
@@ -0,0 +1,12 @@
+config const myString = "test";
```

Comment:
Consider adding a test case with long enough string/bytes to actually trigger a GET